src/style/index.css

```
@@ -0,0 +1,20 @@
+@import "./reset.css";
+@import "./stars.css";
+@import "./card.css";
+
+:root {
+  --color-principal: #060b35;
+  --color-secondary: #616070;
+  --buy-button-principal-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card-page {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  line-height: 14px;
+  font-size: 10px;
+  font-weight: 400;
+}
```

```suggestion
* {
  box-sizing: border-box;
}

.card-page {
  font-family: Roboto, Arial, Helvetica, sans-serif;
  line-height: 14px;
  font-size: 10px;
  font-weight: 400;
}
```
lets create normalize file and move it there